internals/daemon/api_signals_test.go

```diff
@@ -31,6 +31,7 @@ services:
     test1:
         override: replace
         command: sleep 10
+        on-failure: ignore
```

Comment:
Because by default on-failure is restart, in which case, the test case would fail because [it's waiting for the state to be inactive](https://github.com/canonical/pebble/blob/master/internals/daemon/api_signals_test.go#L90).

I gave it some thought before making it `on-failure: ignore`:

- We could add a 1-second sleep or mock the okay wait period, so that when the service is killed, it's not starting, but running, so the autorestart won't happen, but this means either the test would take longer or more complicated.
- We could also change the final part of the test, instead of waiting for it to be in inactive status, it should be backoff. But this highly depends on the okay wait period, especially considering there are some sleeping in the test to wait for the service to be up and running, so the test case might be flaky.

So, I decided make it `on-failure: ignore` so that the test case would always pass without any other change.